deployment/helm/resource-policy/templates/daemonset.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    {{- include "nri-plugins.labels" . | nindent 4 }}
+  name: nri-resource-policy
+  namespace: {{ .Values.namespace }}
```

Hmm, I think we should respect the helm `--namespace` option

```suggestion
  namespace: {{ .Release.Namespace }}
```

Ditto other templates and drop `namespace` from values.yaml(s)

---

Or, actually we probably wouldn't (or shouldn't) need to supply the `namespace:` field in the chart at all. Helm will install stuff in the correct namespace automatically

---

...deleted comment about omitting the `namespace:` field, I think it's cleaner to have it there explicitly